src/plugins/cookies/cookie.js

```diff
@@ -0,0 +1,38 @@
+import URI from 'URIjs';
+
+export default class ChromeCookie {
+
+    constructor (details) {
```

Comment:
I see. But Airbnb style is not compatible with Yandex: e. g. it use indents with 2 spaces. How they work together? )